app/Models/Banner.php

```diff
@@ -114,16 +116,16 @@ public function campaign(): BelongsTo
 
     protected function toArrayExtras($array)
     {
-        $array['serve_url'] = route('banner-serve', ['id' => $this->uuid]);
-        $array['view_url'] = route('banner-view', ['id' => $this->uuid]);
-        $array['click_url'] = route('banner-click', ['id' => $this->uuid]);
+        $array['serve_url'] = $this->removeScheme(route('banner-serve', ['id' => $this->uuid]));
+        $array['view_url'] = $this->removeScheme(route('banner-view', ['id' => $this->uuid]));
+        $array['click_url'] = $this->removeScheme(route('banner-click', ['id' => $this->uuid]));
```

Comment:
I do not see any sense to remove a protocol on the Demand side. I guess that the reason was `image_url` which is not available on the Supply side. 
Should be discussed.